src/app/beer_garden/app.py

```diff
@@ -284,6 +285,10 @@ def _startup(self):
         if config.get("plugin.local.logging.config_file"):
             self.plugin_local_log_config_observer.start()
 
+        if config.get("scheduler.job_startup_file"):
```

Comment:
`if config.get("scheduler.job_startup_file") and os.path.isfile(config.get("scheduler.job_startup_file"):`